README.md

```diff
@@ -1 +1,62 @@
+# API Reference 2.0
 
+##Introduction
+The Open API 2.0 is the new version of the publicly available protobuf-based API developed by Spotware. It allows third-party service providers to integrate additional tools and applications for trading and analysis with the data and functionality from Spotware platform by getting all the required cTID data, market data and performing all possible trading operations on behalf of other cTrader users.
+
+##Documentation
+Full documentation can be found on [Spotware Connect website](https://connect.spotware.com/docs/open_api_2)
+
+## How to start
+1. Download this project.
+2. Import the project to the chosen IDE as a Maven project.
+4. Run command `mvn install` to download libraries and compile Protocol Buffer classes.
+0. Register your application as described [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2). Set `CLIENT_PUBLIC_ID` and `CLIENT_SECRET`
```

Comment:
Specify file where to put those parameters.

Reply:
It is already specified (application.properties)